h/static/scripts/vendor/jquery.scrollintoview.js

```diff
@@ -109,29 +109,37 @@
 
 				var animOptions = {};
 
+				var yDir = "none";
```

Comment:
Let's simply not do this. This is all inane.

Even with all the work you just put into this, the padding does not
consider which element needs to scroll to produce that effect.

I'm still in favor of killing the bucket bar or, at least, the scrolling
feature. I still believe the feature does not serve any use case that so
much needs such a feature that it justifies it's UI weight.

The scrolling has been a constant source of bugs and we've never fully
implemented it to deal with all the cases that present.

I say remove all this scrolling nonsense.